### PR TITLE
ADEN-10592 Cleanup of TCFv1.1 IAB API standard

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/gdpr/ConsentManagementProviderLegacy-test.js [hunk 1]
```diff
@@ -159,8 +159,7 @@ describe('ConsentManagementProvider', () => {
             });
         });
 
-        // ToDo: this test fails only on Jenkins
-        it.skip('implements getVendorConsents with parameters', (done) => {
+        it('implements getVendorConsents with parameters', (done) => {
```
Comment (Loczi94) [inline src/gdpr/ConsentManagementProviderLegacy-test.js:162]: These are legacy tests. Please delete them.

File: src/gdpr/Tracker.js [hunk 2]
```diff
@@ -27,7 +27,6 @@ class Tracker {
             [TRACK_PARAMS.LANGUAGE_CODE]: language,
             [TRACK_PARAMS.DETECTED_GEO]: detectedGeo,
         };
-        this.tcfVersion = 1;
```
Comment (Loczi94) [inline src/gdpr/Tracker.js:30]: Don't we need to track `tcfVersion=2`? 🤔

File: src/index.js [hunk 3]
```diff
@@ -99,62 +89,7 @@ function initializeGDPR(options) {
...
     );
 
-    GeoManager.fetchInstantConfig().then(() => {
-        // ToDo: get rid of it, move Google's id to consts.js
-        if (geoManager.isGoogleMoved()) {
-            enabledVendors.push(
-                755, // Google Advertising Products
-            );
-
-            instance.configure({
```
Comment (Loczi94) [inline src/index.js:109]: Please make sure that it's not needed.

File: src/gdpr/Tracker.js (hunk 2 above)
Comment (dmnsobczak) [reply src/gdpr/Tracker.js:30]: No, TCFv1 is a history now, we will track just events without `tcf-vX` postfix

File: src/gdpr/ConsentManagementProviderLegacy-test.js (hunk 1 above)
Comment (dmnsobczak) [reply src/gdpr/ConsentManagementProviderLegacy-test.js]: Deleted. Restoring moved to another ticket.

File: src/index.js (hunk 3 above)
Comment (dmnsobczak) [reply src/index.js:109]: It's not. Removed.

**Pull request merged**